pkg/recipes/driver/gitconfig_test.go

```diff
@@ -37,34 +36,26 @@ func TestAddConfig(t *testing.T) {
 		{
 			desc:             "success",
 			templatePath:     "git::https://github.com/project/module",
-			expectedResponse: "[url \"https://test-user:ghp_token@github.com\"]\n\tinsteadOf = https://env1-app1-test-redis-recipe-github.com\n",
+			expectedResponse: "[url \"https://test-user:ghp_token@github.com\"]\n\tinsteadOf = https://github.com\n",
```

Comment:
We will need to add some more tests here since there are additions to the logic in the **gitconfig.go** file.